### ISSUE TITLE
Add USB Mass Storage Device for VMU Memory

### ISSUE THREAD

**Issue by Tails86** [OrangeFox86/DreamcastControllerUsbPico#10]

**Comment by hpx1984** [comment]
Hi Sir
how to use this function?

**Comment by Tails86** [comment]
It's an open item. I haven't done any development for it yet, but I
envision setting up a mass storage device with maybe directory structure of
all files found. Or maybe just the raw data files of each VMU which could
then be imported into an emulator.

**Comment by Tails86** [comment]
I'm diving into it now - probably another week or two, depending on how much free time I find. FAT16 drive is setup with placeholders for the raw memory files. It's very likely that these files can be dumped right to/from redream vmu bin files (haven't checked that yet, but the file size of redream files is exactly the size of memory available in a VMU).
![image](https://user-images.githubusercontent.com/12876919/202925002-0ceb5b86-9b0c-4f3b-93ad-fbf235979b7e.png)
![image](https://user-images.githubusercontent.com/12876919/202925027-583f6ba8-8d8f-44f7-8ecf-e4b7a9962523.png)

**Comment by Tails86** [comment]
I tweaked things a bit so that files only show up when the associated memory unit is attached. I confirmed that the file is compatible with redream. Files at the moment are read only.

![image](https://user-images.githubusercontent.com/12876919/203906963-03c04aa1-195a-4db4-b075-1071b551d733.png)
![image](https://user-images.githubusercontent.com/12876919/203906979-1e50ea92-f67c-4456-83d6-91ff171db579.png)
